Formula/s/swig@4.1.rb

```diff
@@ -0,0 +1,55 @@
+class SwigAT41 < Formula
+  desc "Generate scripting interfaces to C/C++ code"
+  homepage "https://www.swig.org/"
+  url "https://downloads.sourceforge.net/project/swig/swig/swig-4.1.1/swig-4.1.1.tar.gz"
```

Comment:
Is 4.1.x release line going to be supported?

Reply:
I am not familiar with the SWIG project, so I do not know.
I could not decide whether it was okay to remove the alias for swig@4.1, so I left it as it is.

Reply:
> I am not familiar with the SWIG project, so I do not know.

In that case just remove the `swig@4.1` formula as versioned formulae need to meet standards listed at: https://docs.brew.sh/Versions#acceptable-versioned-formulae

Reply:
OK, I'll remove it